src/style.css

```diff
@@ -1 +1,121 @@
-/* add styles here */
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url(fonts/Avenir-Book.ttf) format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  font-style: normal;
+  src: url(fonts/Avenir-Heavy.ttf) format('truetype');
+}
+
+.page {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+}
+
+.search-bar {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.search-bar-big {
+  width: 100%;
+  height: 70px;
```

Comment:
Avoid fixing container sizes unless it's a requirement. Here, you're setting a fixed height for `.search-bar-big`. Let the content size dictate the container size to avoid overflow or accidental scroll bars.